pkg/agent/agent_linux.go

```diff
@@ -197,8 +198,9 @@ func (i *Initializer) ConnectUplinkToOVSBridge() error {
 
 	// If uplink is already exists, return.
 	uplink := uplinkNetConfig.Name
-	if _, err := i.ovsBridgeClient.GetOFPort(uplink, false); err == nil {
-		klog.Infof("Uplink %s already exists, skip the configuration", uplink)
+	if uplinkOFPort, err := i.ovsBridgeClient.GetOFPort(uplink, false); err == nil {
+		klog.InfoS("Uplink already exists, skip the configuration", "uplink", uplink, "port", uplinkOFPort)
+		i.nodeConfig.UplinkNetConfig.OFPort = uint32(uplinkOFPort)
```

Comment:
This won't work and make e2e testbed broken, because UplinkNetConfig.OFPort is consumed before this.

Reply:
Great catch, updated.